packages/odyssey-react-mui/src/properties/odyssey-react-mui.properties

```
@@ -39,16 +39,16 @@ table.fetchedrows.text_plural = Fetched {{totalRows}} total rows
 table.moreactions.arialabel = More actions
 table.noresults.heading = There are no results.
 table.noresults.text = Try a different query.
-table.pagination.loadmore = Show more
-table.pagination.next = Next page
-table.pagination.previous = Previous page
-table.pagination.page = Page
-table.pagination.rowsperpage = Rows per page
-table.pagination.rowswithtotal = {{firstRow}}-{{lastRow}} of {{totalRows}} rows
-table.pagination.rowswithouttotal = {{firstRow}}-{{lastRow}} rows
 table.reorder.backward = Send backward
 table.reorder.forward = Bring forward
 table.reorder.toback = Send to back
 table.reorder.tofront = Bring to front
 table.rows.text = {{totalRows}} row
-table.rows.text_plural = {{totalRows}} rows
\ No newline at end of file
+table.rows.text_plural = {{totalRows}} rows
+pagination.loadmore = Show more
+pagination.next = Next page
+pagination.previous = Previous page
+pagination.page = Page
+pagination.rowsperpage = Rows per page
+pagination.rowswithtotal = {{firstRow}}-{{lastRow}} of {{totalRows}} rows
+pagination.rowswithouttotal = {{firstRow}}-{{lastRow}} rows
```

These should probably use underscores:
```suggestion
pagination.rows_with_total = {{firstRow}}-{{lastRow}} of {{totalRows}} rows
pagination.rows_without_total = {{firstRow}}-{{lastRow}} rows
```

---

@KevinGhadyani-Okta None of our other keys use underscores currently. Although I think they should also. This should be a global change if we decide to do this IMHO. Probably handle in another PR